go.mod

```diff
@@ -17,76 +18,39 @@ exclude (
 
 require (
 	github.com/Azure/go-ansiterm v0.0.0-20170629204627-19f72df4d05d // indirect
-	github.com/BurntSushi/toml v0.3.1 // indirect
-	github.com/Microsoft/go-winio v0.4.12 // indirect
-	github.com/Microsoft/hcsshim v0.6.3 // indirect
-	github.com/Microsoft/opengcs v0.3.9 // indirect
+	github.com/Microsoft/go-winio v0.4.13-0.20190408173621-84b4ab48a507 // indirect
 	github.com/Nvveen/Gotty v0.0.0-20120604004816-cd527374f1e5 // indirect
-	github.com/armon/go-radix v1.0.0 // indirect
 	github.com/boltdb/bolt v1.3.1 // indirect
-	github.com/containerd/containerd v0.2.10-0.20170808145631-06b9cb351610 // indirect
+	github.com/containerd/containerd v1.3.0-0.20190507210959-7c1e88399ec0 // indirect
 	github.com/containerd/continuity v0.0.0-20190827140505-75bee3e2ccb6 // indirect
-	github.com/coreos/bbolt v1.3.2 // indirect
-	github.com/coreos/etcd v3.3.13+incompatible // indirect
-	github.com/coreos/go-semver v0.3.0 // indirect
-	github.com/coreos/go-systemd v0.0.0-20190719114852-fd7a80b32e1f // indirect
-	github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f // indirect
-	github.com/deckarep/golang-set v1.7.1 // indirect
-	github.com/dgrijalva/jwt-go v3.2.0+incompatible // indirect
-	github.com/docker/distribution v2.6.0-rc.1.0.20170726174610-edc3ab29cdff+incompatible
-	github.com/docker/docker v0.0.0-20190404075923-dbe4a30928d4
+	github.com/containerd/fifo v0.0.0-20190816180239-bda0ff6ed73c // indirect
+	github.com/containers/image/v5 v5.0.0
```

Comment:
this is ok - containers/image uses strict semvers so we won't have transitive dep conflicts in openshift/builder